.babelrc

```diff
@@ -1,9 +1,27 @@
 {
     "presets": [
-        "@wordpress/babel-preset-default"
```

Comment:
Actually we'll need a separate babel config, because this overrides the [default Wordpress setup](https://github.com/WordPress/gutenberg/blob/HEAD/packages/babel-preset-default/index.js#L61), which we need in the editor.